# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/cairo/cairo/src/cairo-spans.c
# Commit: 3cdfb39376b7
# Full Hash: 3cdfb39376b736597fe658a322389117686b7e82
# Author: Jonathan Kew <jkew@mozilla.com>
# Date: 2024-05-07 09:13:07
# Regressor Bug: 1892913
# File Overlap Count: 3
# Description:
#   Bug 1892913 - patch 1 - Update cairo source to 1.18.0 r=gfx-reviewers,lsalzman
#   
#   Wholesale replacement of files in gfx/cairo/cairo with contents of the 1.18.0 release tarball,
#   omitting subdirectories that are not part of the core library build.
#   
# ==============================================================================

diff -r 9daa351f875b -r 3cdfb39376b7 gfx/cairo/cairo/src/cairo-spans.c
--- a/gfx/cairo/cairo/src/cairo-spans.c	Mon May 06 19:22:35 2024 +0000
+++ b/gfx/cairo/cairo/src/cairo-spans.c	Mon May 06 19:25:12 2024 +0000
@@ -132,6 +132,8 @@
     case CAIRO_STATUS_FREETYPE_ERROR:
     case CAIRO_STATUS_WIN32_GDI_ERROR:
     case CAIRO_STATUS_TAG_ERROR:
+    case CAIRO_STATUS_DWRITE_ERROR:
+    case CAIRO_STATUS_SVG_FONT_ERROR:
     default:
 	break;
     }
@@ -249,6 +251,8 @@
     case CAIRO_STATUS_FREETYPE_ERROR: RETURN_NIL;
     case CAIRO_STATUS_WIN32_GDI_ERROR: RETURN_NIL;
     case CAIRO_STATUS_TAG_ERROR: RETURN_NIL;
+    case CAIRO_STATUS_DWRITE_ERROR: RETURN_NIL;
+    case CAIRO_STATUS_SVG_FONT_ERROR: RETURN_NIL;
     default:
 	break;
     }